fetch.py

```diff
@@ -466,7 +480,7 @@ def configure_logging(log_file: str) -> None:
     """
 
 
-    logger.setLevel(logging.DEBUG)
+    logger.setLevel(logging.INFO)
 
     formatter = logging.Formatter(
         '%(asctime)s %(name)-12s [PID:%(process)d] %(levelname)-8s %(message)s'
```

Comment:
number of spaces between name and process id should be tailored so that there are not too many spaces without a reason